pkg/koordlet/statesinformer/states_informer.go

```diff
@@ -209,3 +226,29 @@ func (s *statesInformer) setupInformers() {
 	s.setupNodeInformer()
 	s.setupNodeSLOInformer()
 }
+
+func newKubeletStub(node *corev1.Node, addressPreferred string, timeout int) (KubeletStub, error) {
+	var address string
+	var err error
+	addressPreferredType := corev1.NodeAddressType(addressPreferred)
+	// if the address of the specified type has not been set or error type, InternalIP will be used.
+	if addressPreferredType == "" ||
```

Comment:
could you please improve the readability of the statement? e.g. `if IsAddressTypeSupported()`